core/domain/rights_manager.py

```diff
@@ -252,21 +344,59 @@ def get_exploration_rights(exploration_id, strict=True):
 
 
 def is_exploration_private(exploration_id):
+    """Returns whether exploration is private.
+
+    Args:
+        exploration_id: str. ID of exploration.
+
+    Returns:
+        bool. Whether the exploration is private or not.
+    """
     exploration_rights = get_exploration_rights(exploration_id)
     return exploration_rights.status == ACTIVITY_STATUS_PRIVATE
 
 
 def is_exploration_public(exploration_id):
+    """Returns whether exploration is public.
+
+    Args:
+        exploration_id: str. ID of exploration.
+
+    Returns:
+        bool. Whether the exploration is public.
+    """
     exploration_rights = get_exploration_rights(exploration_id)
     return exploration_rights.status == ACTIVITY_STATUS_PUBLIC
 
 
 def is_exploration_cloned(exploration_id):
+    """Returns whether exploration is cloned by someone else.
+
+    Args:
+        exploration_id: str. ID of exploration.
+
+    Returns:
+        bool. Whether the exploration is cloned.
+    """
     exploration_rights = get_exploration_rights(exploration_id)
     return bool(exploration_rights.cloned_from)
 
 
 def create_new_collection_rights(collection_id, committer_id):
+    """Creates a new collection for the committer.
+
+    Creates a default rights object for the collection.
```

Comment:
See previous comments for the corresponding exploration-related function; similar comments apply here.